packages/web/src/components/gcds-grid/gcds-grid-col.tsx

```diff
@@ -0,0 +1,86 @@
+import { Component, Element, Host, Watch, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'gcds-grid-col',
+  styleUrl: 'gcds-grid-col.css',
+  shadow: true,
+})
+export class GcdsGridCol {
+  @Element() el: HTMLElement;
+
+  /**
+   * Props
+   */
+
+  /**
+   * Set tag for grid column
+   */
+  @Prop() tag?: string = 'div';
+
+  /**
+   * Optimize grid column size for tablet (768px - 1023px).
+   * Tablet grid column sizes are based on a 6 column grid.
+   * The tablet size will also be used for desktop, if desktop is undefined.
+   */
+  @Prop() tablet?: 1 | 2 | 3 | 4 | 5 | 6 = 6;
```

Comment:
Since the component has a chance to change this value itself, we will need to set `@Prop({ mutable: true })` to allow it.

Reply:
Good catch!